sefaria/urls.py

```diff
@@ -385,7 +390,8 @@
 
 # Email Subscribe
 urlpatterns += [
-    url(r'^api/subscribe/(?P<email>.+)$', sefaria_views.subscribe),
+    url(r'^api/subscribe/steinsaltz/(?P<email>.+)$', sefaria_views.subscribe_steinsaltz_newsletter_view),
```

Comment:
I think we can probably generalize this. I dont think we need to have a dedicated view specifically for Steinsaltz.  We can have a general view for a foreign newsletter and perform the routing based on parameter to the correct logic if needed.

Reply:
Did you have in mind that also the Sefaria newsletter would be generalized so there's only one subscribe view?

Reply:
Not necessarily, but it can be (i.e. waht the view defaults to if no parameter is passed in)